### ISSUE TITLE
Custom Regularizers

### ISSUE THREAD

**Issue by alisonbma** [KevinMusgrave/pytorch-metric-learning#456]
I was wondering if there would be a way for us to
(i) Modify existing pytorch_metric_library regularizers and have the regularizer accept user hyperparameters? This would be great also if the compute_loss function could accept new regularizer hyperparameters for various training epochs
(ii) Use pytorch_metric_learning regularizers without reduction?

**Comment by KevinMusgrave** [comment]
Let me know if you have more questions.
